suite-common/connect-init/src/connectInitThunks.ts

```diff
@@ -108,11 +108,18 @@ export const connectInitThunk = createThunk(
 
         cardanoConnectPatch(getEnabledNetworks);
 
+        const sessionsBackground =
+            window.location.origin +
+            `${process.env.ASSET_PREFIX || ''}/static/connect/workers/sessions-background-sharedworker.js`.replaceAll(
+                /\/+/g,
+                '/',
+            );
```

Comment:
by using a `_` prefixed variable I am trying to express that although this is part of public API, you are not supposed to use it. 
 
I was also thinking about other options how to achieve the same without introducing a new connect init param, such as setting the URL somewhere deeper using env variable but it felt even clumsier than this.